static/app/components/forms/controls/textarea.tsx

```diff
@@ -1,13 +1,16 @@
 import {forwardRef} from 'react';
-import TextareaAutosize from 'react-autosize-textarea';
+import TextareaAutosize, {type TextareaAutosizeProps} from 'react-textarea-autosize';
 import isPropValid from '@emotion/is-prop-valid';
 import styled from '@emotion/styled';
 
 import type {InputStylesProps} from 'sentry/components/input';
 import {inputStyles} from 'sentry/components/input';
 
 export interface TextAreaProps
-  extends Omit<React.TextareaHTMLAttributes<HTMLTextAreaElement>, 'css' | 'onResize'>,
+  extends Omit<
+      React.TextareaHTMLAttributes<HTMLTextAreaElement>,
+      'css' | 'onResize' | 'height' | 'style'
```

Comment:
I understood why we are omitting 'style' here, but do we need to omit 'height' too? it seems it is not part of the types.